content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.getActualMinimum()'
+Description: 'Returns the minimum value that the specified calendar field could have based on the time value of this calendar'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Time'
```

Comment:
```suggestion
  - 'Calendar'
  - 'Time'
  - 'Date'
```

Reply:
Thank you for the suggestions. I've made and committed the changes. Let me know if I did something wrong or if the changes don't show for some reason.